src/MetricsUploader/MetricsUploaderWindowsTest.cpp

```diff
@@ -7,65 +7,66 @@
 #include <chrono>
 
 #include "MetricsUploader/MetricsUploader.h"
+#include "MetricsUploader/MetricsUploaderStub.h"
 #include "OrbitBase/Result.h"
 #include "absl/container/flat_hash_set.h"
 #include "absl/strings/ascii.h"
 
 namespace orbit_metrics_uploader {
 
+using orbit_metrics_uploader::MetricsUploaderStub;
+
 TEST(MetricsUploader, CreateMetricsUploaderFromClientWithoutSendEvent) {
   auto metrics_uploader =
       MetricsUploader::CreateMetricsUploader("MetricsUploaderClientWithoutSendEvent");
-  EXPECT_FALSE(metrics_uploader.has_value());
+  EXPECT_NE(dynamic_cast<MetricsUploaderStub*>(metrics_uploader.get()), nullptr);
```

Comment:
Nit: On first sight it's a bit hard to understand what's happening here. This would benefit from separating the check into its own function `[[nodiscard]] static bool IsMetricsUploaderStub(const std::unique_ptr<MetricsUploader>&)`

Reply:
Agree, I've moved this logic to the separate function in this test file, thank you!.